apps/session-manager/api/pagopa.yaml

```diff
@@ -0,0 +1,86 @@
+swagger: "2.0"
+info:
+  version: 13.38.0
+  title: Proxy API
+  description: Mobile and web proxy API gateway.
+host: localhost
+basePath: /pagopa/api/v1
+schemes:
+  - https
+security:
+  - Token: []
+paths:
+  /user:
+    get:
+      operationId: getUser
+      summary: Get user's data
+      description: Returns the user data needed by PagoPA Wallet backend.
+      parameters:
+        - in: query
+          name: version
+          type: string
+          enum:
+            - "20200114"
+      responses:
+        "200":
+          description: Found.
+          schema:
+            $ref: "#/definitions/PagoPAUser"
+          examples:
+            application/json:
+              email: email@example.com
+              name: Name
+              family_name: Surname
+        "400":
+          description: Bad request.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Token null or invalid.
+        "404":
+          description: User Profile not found
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "429":
+          description: Too many requests
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "500":
+          description: There was an error in retrieving the user data.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+definitions:
+  EmailAddress:
+    type: string
+    format: email
+    example: foobar@example.com
+  ProblemJson:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v28.18.3/openapi/definitions.yaml#/ProblemJson"
+  FiscalCode:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v28.18.3/openapi/definitions.yaml#/FiscalCode"
+  FederatedUser:
+    $ref: "internal.yaml#/definitions/FederatedUser"
+  PagoPAUser:
+    title: PagoPA user
+    description: User data needed by PagaPA proxy.
+    allOf:
+      - $ref: "#/definitions/FederatedUser"
+      - type: object
+        properties:
+          spid_email:
+            $ref: "#/definitions/EmailAddress"
+          notice_email:
+            $ref: "#/definitions/EmailAddress"
+        required:
+          - notice_email
+responses: {}
+parameters: {}
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  Token:
+    type: apiKey
+    name: token
+    in: query
```

Comment:
Maybe there is an error on this definition. Looking the library used to extract the token from the request, seems that `access_token` is the right query prop.

_from `passport_http_bearer/lib/strategy.js`:_
```javascript
if (req.query && req.query.access_token) {
  if (token) { return this.fail(400); }
  token = req.query.access_token;
}
```
The library support header, body and query params to transport the token, we should consider to declare in OpenAPI a more safe way, like header.

Reply:
addressed in https://github.com/pagopa/io-auth-n-identity-domain/pull/55/commits/2a7137aa0ebbcac12991c1d833edfbf2bbb2e39d